lib/middleware/index.js

```diff
@@ -3,29 +3,28 @@
  * Attach middleware and such to the app.
  */
 
-var env = process.env.NODE_ENV || 'development';
+const env = process.env.NODE_ENV || 'development';
+const convert = require('koa-convert');
 
 module.exports = function (app, options) {
   options = options || {};
 
   // methods
   if (options.responseTime !== false) app.use(require('koa-response-time')());
-  app.use(require('./trace'));
+  // app.use(require('./trace'));
```

Comment:
Did you want to use [koa-trace](https://www.npmjs.com/package/koa-trace) for this? I noticed you have a fork of it.